### PR TITLE
Adds changes to prep package for submission to python package library

### PR DESCRIPTION
**Pull request by ezmiller**
NOT READY TO BE MERGED. Open for discussion.

### CODE REVIEW

**Comment by ezmiller** [comment]
@ligthyear question on this: is there a way (and a need) to specify that this module supports only Python 3 (which I think is what we decided)?

**Comment by ligthyear** [comment]
@ezmiller not sure. [I think you only add the proper tags](https://docs.python.org/3.3/howto/pyporting.html#before-you-begin) and don't add the python-2 ones. But even if not, the import would fail on any python2 kinda immediately.

I'd note it at the end of the Readme and Description of the package, so if someone researches it, they'll know before installing that.

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2010 by Armin Ronacher.
+Copyright (c) 2016 by Ethan Miller.
```
Comment (ligthyear) [inline LICENSE]: It's custom to still mention the original author. As in "based on the Flask-Babel Package by Armin Ronacher".

File: setup.py [hunk 2]
```diff
@@ -19,11 +19,11 @@
 
 setup(
     name='Flask-ICU',
-    version='0.9',
+    version='0.1',
```
Comment (ligthyear) [inline setup.py]: I think we are further no? Consider what would you expect a `1.0` to contain and then substract `0.1` for each big feature missing. I think we are closer to `0.8` or `0.9` than `0.1` – we at least have an API we believe will be close to `1.0`. Don't sell yourself short!

File: setup.py [hunk 3]
```diff
@@ -35,7 +35,7 @@
         'speaklater>=1.2',
     ],
     classifiers=[
-        'Development Status :: 4 - Beta',  # TODO: update dev status
+        'Development Status :: 4 - Beta',
```
Comment (ligthyear) [inline setup.py]: We should add `'Programming Language :: Python :: 3'` here - and no `:: 2` to send the clear message of support.

Comment (ezmiller) [inline setup.py]: This tag already exists just below.

**Pull request merged**